postgis_helpers.go

```diff
@@ -0,0 +1,111 @@
+package tilenol
+
+import (
+	"bytes"
+	"database/sql"
+	"time"
+
+	"github.com/paulmach/orb"
+	"github.com/paulmach/orb/encoding/wkb"
+)
+
+// RowsToMaps converts Go's sql.Rows data structure into a list of maps where the key is a string
+// column name, and the value is the raw SQL value
+func RowsToMaps(rows *sql.Rows) ([]map[string]interface{}, error) {
+	var maps []map[string]interface{}
+
+	cols, err := rows.Columns()
+	if err != nil {
+		return maps, err
+	}
+
+	for rows.Next() {
+		row := make([]interface{}, len(cols))
+		for idx := range cols {
+			row[idx] = new(RowScanner)
+		}
+		err := rows.Scan(row...)
+		if err != nil {
+			return maps, err
+		}
+		m := make(map[string]interface{})
+		for idx, col := range cols {
+			var s = row[idx].(*RowScanner)
+			if s.valid {
```

Comment:
My guess is this means that a missing value (say, we are asking for `name` for each feature) means that the corresponding key for that value would be absent from that feature? (as opposed to pointing to a `null` value)? 

Just want to confirm this is consistent with what the ES backend currently does?

Reply:
It's actually not so much that the value is missing in the database response, but rather that the value that came back from the database just cannot be dealt with by our code.

To clarify: at this point in the code, the `row[idx].Scan(src)` method has already been called implicitly (in line 27), and so it's internal state of `valid` and `value` should be set by the big `switch` statement starting on line 72. So the only time that `valid` is set to `false` is when we miss all of the type-handler `case`s.